packages/core/scripts/Voting.js

```diff
@@ -261,36 +259,6 @@ async function fetchIntrinioForexPrice(request, config, isProd) {
   return web3.utils.toWei(price.toString());
 }
 
-async function fetchIntrinioCryptoPrice(request, config, isProd) {
```

Comment:
RIP, but also beautiful to see this removed

Reply:
There are other intrinio query methods still there -- it was just the crypto one that wasn't being used.